dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -86,23 +87,26 @@ private static Repository createClassRepository(List<Path> paths) {
   static ClassDumper create(List<Path> jarPaths) throws IOException {
     ClassLoader systemClassLoader = ClassLoader.getSystemClassLoader();
     ClassLoader extensionClassLoader = systemClassLoader.getParent();
+    jarPaths.forEach(
+        jar -> {
+          checkArgument(Files.isRegularFile(jar), "The jar file is not a regular file: %s", jar);
```

Comment:
Previously, when an invalid input was given, it was caught at this function. But it's no longer the case.
https://github.com/GoogleCloudPlatform/cloud-opensource-java/pull/496/files#diff-6e13c859770f686d992e68ec76956e75R103
Without `checkArgument`, `ClassDumper.create()` would succeed even with invalid input.